core/foundation/src/TClassEdit.cxx

```diff
@@ -1342,7 +1342,7 @@ bool TClassEdit::IsStdClass(const char *classname)
 bool TClassEdit::IsVectorBool(const char *name) {
    TSplitType splitname( name );
 
-   return ( TClassEdit::STLKind( splitname.fElements[0] ) == ROOT::kSTLvector)
+   return ( TClassEdit::STLKind( std::string_view(splitname.fElements[0].data(), splitname.fElements[0].size()) ) == ROOT::kSTLvector)
```

Comment:
Given how often we need to handle the `STLKind` constructor, maybe you could add an overload taking a `string_view` (if the disambiguation is really needed)?